pkg/selector/outputs/bubbletea.go

```diff
@@ -57,17 +72,36 @@ var (
 	}
 )
 
+// verboseState represents the current state of the verbose view
+type verboseState struct {
```

Comment:
Can we rename this to `verboseModel` to align with the other `tableModel` and `BubbleTeaModel` that we use in similar ways?